content/id/docs/tutorials/hello-minikube.md

```diff
@@ -249,13 +249,13 @@ kubectl delete service hello-node
 kubectl delete deployment hello-node
 ```
 
-Kamu juga boleh mematikan mesin virtual (VM) untuk Minikube:
+Kamu juga boleh mematikan mesin virtual (VM) untuk minikube:
```

Comment:
I don't have a strong opinion on this, but VM is actually abbreviation of _virtual machine_. Thus, imo abbreviating _mesin virtual_ with VM is a bit weird.

Reply:
I put like this `mesin virtual atau virtual machine (VM)`